### DISCUSSION TITLE
BIRT exports xlsx data in one 'Sheet' without multiple 'Sheets'

### DISCUSSION THREAD

**Discussion by 858882847** [eclipse-birt/birt#1241] [Q&A]
[Uploading test.zip…]()

**Comment by speckyspooky** [comment]
An option is the user property at report level with the name ExcelEmitter.SingleSheet -> Type: Boolean -> Value: True (#1240)
(Your link doesn't work)

**Reply by 858882847** [reply]
Hello speckyspooky "I'm not quite sure where to set the option 'ExcelEmitter.SingleSheet -> Type: Boolean -> Value: True' mentioned. Could you please provide a screenshot for me as a reference? Thank you."

**Reply by 858882847** [reply]
![I9UJMC46~J)DKP6 A 4WN$M](https://user-images.githubusercontent.com/117696273/227086457-2350a84a-cdfa-420a-a7a2-a31b86d94c30.png)
![OMF35EP_BJO52T((Q2O4RPQ](https://user-images.githubusercontent.com/117696273/227086463-d8b84b0b-681d-4944-a55b-62bc8679d12d.png)

**Reply by 858882847** [reply]
"I set it in the outermost table and it didn't work when I set it inside, but you said the repeated headers that should be hidden must be dealt with. I don't quite understand what you mean."

**Reply by speckyspooky** [reply]
You haven't it to set on table level, this is an user property based on report level (root).
Please mark the report element and add the user property there:

![ExcelEmitter SingleSheet](https://user-images.githubusercontent.com/41593722/227331304-10656dab-072f-448e-b685-45662f916ec7.png)

**Comment by NWesW** [comment]
Try adding this to the header, onRender 
`if( reportContext.getOutputFormat() == "xlsx"){
if(pageNumber>1)
{
this.getStyle().display="none";
}
}
beforeFactory
`if(params["XLSX"].value == 1){
reportContext.getAppContext().put("MAX_PAGE_BREAK_INTERVAL",1000000);
} else {
var table = reportContext.getReportRunnable().designHandle.getDesignHandle().findElement("main");
table.setProperty("pageBreakInterval", 40);
}

 `
Then set the Page Break Interval under properties tab Page Break to 1000000

Note I added a parameter which may not be necessary for your reports.
See if this works for you
